lib/cobra_commander/component_tree.rb

```diff
@@ -64,13 +64,20 @@ def initialize(root_path)
         def dependencies
           @deps ||= begin
             return [] unless gem?
-            gems = bundler_definition.dependencies.select do |dep|
-              dep.source&.is_a_path? && dep.source.path.to_s != "."
-            end
+            gems = bundler_definition.dependencies.select { |dep| path?(dep.source) }
             format(gems)
           end
         end
 
+        def path?(source)
+          return if source.nil?
+          if source.respond_to?(:path?)
+            source.path? && source.path.to_s != "."
```

Comment:
Last nit: any chance of reducing the duplication here?

Reply:
I'll see what I can do. I was thinking that if `source` does respond to `.path?`, it may still return  falsy.  And if it does not return truthy, then it may not respond to `path` (without the `?`).